### PR TITLE
warzone: 3.2.3 -> 3.3.0

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
The latest stable is a bit old. Is it ok to update to a beta?
EDIT: the new stable came out yesterday, did an update.

Also this PR wraps the application for #65399.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (is wrapped and runs with `nix-shell --pure -p --run`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @astsmtl

### CODE REVIEW

**Comment by rnhmjoj** [comment]
@GrahamcOfBorg build warzone2100

**Comment by rnhmjoj** [comment]
@GrahamcOfBorg build warzone2100

**Comment by rnhmjoj** [comment]
@GrahamcOfBorg build warzone2100

**Comment by rnhmjoj** [comment]
The url has changed again, hopefully the last time.

**Comment by rnhmjoj** [comment]
@GrahamcOfBorg build warzone2100

File: pkgs/games/warzone2100/default.nix [hunk 1]
```diff
@@ -11,17 +12,25 @@ let
   };
 in
 
-stdenv.mkDerivation rec {
-  version = "3.2.3";
-  name = "${pname}-${version}";
+mkDerivation rec {
+  inherit pname;
+  version  = "3.3.0";
+  checkout = "20190831-182824-924bf1e";
```
Comment (Mic92) [inline pkgs/games/warzone2100/default.nix]: Is this actually used?

Comment (rnhmjoj) [reply pkgs/games/warzone2100/default.nix]: yeah, it's a leftover from the previous source url.

**Comment by rnhmjoj** [comment]
@Mic92 Can you cherry pick this in 19.09 or should I open another PR? I think it's currently broken for missing some Qt library.

**Comment by Mic92** [comment]
[detached HEAD 76ef3295909] warzone: 3.2.3 -> 3.3.0_beta1
 Author: rnhmjoj <rnhmjoj@inventati.org>
 Date: Mon Aug 12 02:38:33 2019 +0200
 1 file changed, 20 insertions(+), 10 deletions(-)
Press Enter to continue
[detached HEAD 75ba6bb7e49] warzone: 3.3.0_beta1 -> 3.3.0
 Author: rnhmjoj <rnhmjoj@inventati.org>
 Date: Mon Sep 2 13:42:47 2019 +0200
 1 file changed, 4 insertions(+), 5 deletions(-)

**Comment by rnhmjoj** [comment]
Thank you!

**Pull request merged**